app/services/import_polygons.rb

```diff
@@ -0,0 +1,40 @@
+class ImportPolygons
+  attr_reader :location_type
+
+  def initialize(location_type:)
+    @location_type = location_type
+  end
+
+  def call
+    response = HTTParty.get(LOCATION_POLYGON_SETTINGS[location_type][:api])
+
+    response.fetch('features', []).each do |region_response|
+      region_name = region_response.dig('attributes', LOCATION_POLYGON_SETTINGS[location_type][:name_key]).downcase
+
+      next unless location_categories_include?(region_name)
+
+      geometry_rings = region_response.dig('geometry', 'rings')
+
+      # The first ring tends to contain far more points than subsequent rings.
```

Comment:
I know I wrote this initially, but my understanding/assumption now is that the first ring is the outer boundary, and that subsequent rings are bodies of water (essentially exclusion rings) that we don't need to care about. So it might be helpful to update the comment to reflect that